docs/en/observability/logs-ecs-application.asciidoc

```diff
@@ -0,0 +1,206 @@
+
+[discrete]
+[[ingest-ecs-logs-with-filebeat]]
+=== Ingest logs with {filebeat}
+
+Follow these steps to ingest application logs with {filebeat}.
+
+[discrete]
+[[step-1-ecs-install-filebeat]]
+=== Step 1: Install {filebeat}
+
+Install {filebeat} on the server you want to monitor by running the commands that align with your system:
+
+include::tab-widgets/logs/filebeat-install-widget.asciidoc[]
+
+[discrete]
+[[step-2-ecs-connect-to-your-project]]
+=== Step 2: Connect to your project
+
+Connect to your project using an API key to set up {filebeat}. Set the following information in the `filebeat.yml` file:
+
+[source,yaml]
+----
+output.elasticsearch:
+  hosts: ["your-projects-elasticsearch-endpoint"]
+  api_key: "id:api_key"
+----
+
+. Set the `hosts` to your deployment's {es} endpoint. Copy the {es} endpoint from your deployment's page and add the port (the default port is `443`). For example, `https://my-deployment.es.us-central1.gcp.cloud.es.io:443`.
++
+--
+[role="screenshot"]
+image::images/es-endpoint-cluster-id.png[{es} endpoint and cluster id location, 50%]
+--
+. From **Developer tools**, run the following command to create an API key that grants `manage` permissions for the `cluster` and the `filebeat-*` indices using:
++
+[source,console]
+----
+POST /_security/api_key
+{
+  "name": "filebeat_host001",
+  "role_descriptors": {
+    "filebeat_writer": {
+      "cluster": ["manage"],
+      "index": [
+        {
+          "names": ["filebeat-*"],
+          "privileges": ["manage"]
+        }
+      ]
+    }
+  }
+}
+----
++
+Refer to {filebeat-ref}/beats-api-keys.html[Grant access using API keys] for more information.
+
+[discrete]
+[[step-3-ecs-configure-filebeat]]
+=== Step 3: Configure {filebeat}
+
+Add the following configuration to your `filebeat.yaml` file to start collecting log data.
+
+[source,yaml]
+----
+filebeat.inputs:
+- type: filestream  <1>
+  enabled: true
+  paths: /path/to/logs.log  <2>
+----
+<1> Reads lines from an active log file.
+<2> Paths that you want {filebeat} to crawl and fetch logs from.
+
+[discrete]
+[[step-4-ecs-set-up-and-start-filebeat]]
+=== Step 4: Set up and start {filebeat}
+
+From the {filebeat} installation directory, set the {ref}/index-templates.html[index template] by running the command that aligns with your system:
+
+include::tab-widgets/logs/filebeat-setup-widget.asciidoc[]
+
+From the {filebeat} installation directory, start filebeat by running the command that aligns with your system:
+
+include::{beats-root}/libbeat/docs/tab-widgets/start-widget-filebeat.asciidoc[]
+
+[discrete]
+[[ingest-ecs-logs-with-agent]]
+=== Ingest logs with {agent}
+
+Add the custom logs integration to ingest and centrally manage your logs using {agent} and {fleet}:
+
+[discrete]
+[[step-1-add-the-custom-logs-integration-to-your-project-ecs]]
+=== Add the custom logs integration to your project
```

Comment:
There are some serverless steps in here that are not true for classic, where projects and project settings don't exist.